### PR TITLE
Fixing broken links to quickstarts and samples

### PR DESCRIPTION
**Pull request by erhopf**
## Purpose

* This change fixes broken links to quickstarts and sample code in the README.md

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,22 +86,22 @@ If you want to build them from scratch, please follow the articles in the Quicks
...
+| [Quickstart C++ for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/cpp/windows/from-microphone) | Windows  | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C++ for macOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/cpp/macos/from-microphone) | macOS |  | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C# .NET for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/dotnet/from-microphone) | Windows | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C# .NET Core](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/dotnetcore/from-microphone) | Windows, Linux, macOS | Demonstrates one-shot speech recognition from a microphone. |
 | [Quickstart C# UWP for Windows](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp-uwp/) | Windows | Demonstrates one-shot speech recognition from a microphone. |
```
Comment (BrianMouncer) [inline README.md]: >(https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp-uwp/) [](start = 33, length = 99)

still broken?

File: README.md [hunk 2]
```diff
@@ -86,22 +86,22 @@ If you want to build them from scratch, please follow the articles in the Quicks
...
+| [Quickstart Python](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/python/from-microphone) | Windows, Linux, macOS | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart Objective-C iOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec/ios/from-microphone) | iOS | Demonstrates one-shot speech recognition from a file with recorded speech. |
+| [Quickstart Swift iOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift/ios/from-microphone) | iOS | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart Objective-C macOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec/macos/from-microphone) | macOS | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart Swift macOS](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift/macOS/from-microphone) | macOS | Demonstrates one-shot speech recognition from a microphone. |
```
Comment (BrianMouncer) [inline README.md]: >https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift/macOS/from-microphone [](start = 27, length = 113)

new link is returning 404 for me..

**Comment by erhopf** [comment]
Fixed the casing issue in the latest push.

File: README.md (hunk 1 above)
Comment (BrianMouncer) [reply README.md]: still, still broken.

I'm not sure what this use to point to, but I'm guessing it should be this one????

[https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone)


---
In reply to: [345453295](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/431#discussion_r345453295) [](ancestors = 345453295)

File: README.md [hunk 3]
```diff
@@ -86,22 +86,22 @@ If you want to build them from scratch, please follow the articles in the Quicks
...
-| [Quickstart Objective-C](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec-ios/) | iOS | Demonstrates one-shot speech recognition from a file with recorded speech. |
-| [Quickstart Swift](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift-ios/) | iOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Objective-C](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/objectivec-macos/) | macOS | Demonstrates one-shot speech recognition from a microphone. |
-| [Quickstart Swift](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/swift-macos/) | macOS | Demonstrates one-shot speech recognition from a microphone. |
+| [Quickstart C# Unity (Windows or Android)](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone) | Windows, Android | Demonstrates one-shot speech recognition from a microphone. |
```
Comment (BrianMouncer) [inline README.md]: >[Quickstart C# Unity (Windows or Android)](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/uwp/from-microphone) [](start = 2, length = 156)

This says it is for UNTY, but is pointing to the UWP sample.

I think this should be this one?

[https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/unity/from-microphone](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/quickstart/csharp/unity/from-microphone)

**Comment by erhopf** [comment]
Fixed both.

File: README.md (hunk 3 above)
Comment (erhopf) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (erhopf) [reply README.md]: fixed.

**Pull request merged**